apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -1162,6 +1175,12 @@ class TrygdetidServiceImpl(
                 trygdetidIdenter.containsAll(avdoede.map { it.value })
         )
     }
+
+    private fun oppdaterBeregnetTrygdetidVedKopieringEnabled() =
```

Comment:
Foreslår å bare flytte denne funksjonen inn der den brukes i stedet, denne funksjonen fjerner litt lesbarhet for meg i alle fall